apps/cookbook/src/app/examples/menu-example/examples/advanced.ts

```diff
@@ -3,10 +3,21 @@ import { Component } from '@angular/core';
 const config = {
   selector: 'cookbook-menu-advanced-example',
   template: `<kirby-menu [closeOnSelect]="false">
-  <kirby-item (click)="actionClicked()">
+  <kirby-item selectable="true">
+    <kirby-icon name="person" slot="start"></kirby-icon>
+    <kirby-checkbox slot="end">
+      <kirby-label>      
+        <h3>Face-id</h3>
+      </kirby-label>
+    </kirby-checkbox>
+  </kirby-item>
+  <kirby-item selectable="true">
     <kirby-icon name="notification" slot="start"></kirby-icon>
-    <h3>Title</h3>
-    <kirby-toggle slot="end" checked="true" (checkedChange)="toggled()"></kirby-toggle>
+    <kirby-checkbox slot="end">
+      <kirby-label>      
+        <h3>Notifications</h3>
```

Comment:
Consider not using an html heading for this:
```suggestion
        Notifications
```